docs/Installation.md

```diff
@@ -1,265 +1,255 @@
+        sudo mv /etc/apache2/sites-available/000-default.conf /etc/apache2/sites-available/z-000-default.conf
+
+#### Create new user and group for apache
+
+        sudo useradd apache
+        sudo groupadd apache
+
+#### Set appropriate folder and file permission on /opt/software-discovery-tool/ folder for apache
+
+        sudo chown -R apache:apache /opt/software-discovery-tool/
+
+#### Start/Restart Apache service
+
+        sudo apachectl restart
+
+### Step 5: Cloning Data Directory (Only First Time)
+ openmainframeproject/software-discovery-tool-data contains all OMP created json files. To add the data files, we will use `git submodule`
+- map the submodule directory with the directory path and update the directory:
+```
+git submodule update --init --recursive --remote
+```
+- since we cloned a new repo as root just now, give the permissions to `apache` user like we did before:
+```
+sudo chown -R apache:apache /opt/software-discovery-tool/distro_data
+```
+
+#### Updating Data Directory
+Everytime there's an upstream change in the submodule:
+- To update the data directory with the main repo with the remote changes:
+```
+sudo -u apache git pull <upstream remote> <default branch> --recurse-submodules
+```
+- To update ONLY the data directory keeping the main repo as it is:
+```
+sudo -u apache git submodule update --recursive --remote
+```
+
+#### Using data from PDS
+For example, taking RHEL_8_Package_List.json
+- Usage help will be displayed:
+```
+cd /opt/software-discovery-tool/bin
+./package_build.py
+Usage:
+./package_build.py <exact_file_name.json>
+			[if data is from PDS]
+./package_build.py debian
+			[if data is from Debian]
+./package_build.py
+			[for displaying this help]
+Example:
+./package_build.py RHEL_8_Package_List.json
+```
+Example of extracting the RHEL_8_Package_List.json from PDS repo:
+```
+./package_build.py RHEL_8_Package_List.json
+Extracting RHEL_8_Package_List.json from PDS data ...
+Thanks for using SDT!
+```
+ _**NOTE:**_
+- Make sure the json file exists in the PDS data directory.
+- Please keep in mind that the directory belongs to user `apache`, so in case of permission error,
+	run the chown cmd or directly use `package_build.py` as `apache` user like:
+	```
+	sudo -u apache ./bin/package_build.py RHEL_8_Package_List.json
+	```
+Now to know how to update the `src/config/supported_distros.py` to reflect the new json files in the UI, follow steps mentioned in
+Step 2 of
+[Adding_new_distros](https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Adding_new_distros.md#step-2-update-the-supported_distros-variable-in-configuration-file-sdt_basesrcconfigconfigpy)
+
+### Step 6: Install and populate the SQL database
+
+#### Install dependencies and complete the secure installation. Remember the root password you set, you will need this in the future.
+
+        sudo apt install mariadb-server python3-pymysql
+        sudo mysql_secure_installation

```

Comment:
FYI I had problems running `mariadb` due to https://mariadb.com/kb/en/authentication-plugin-unix-socket/.  I got around this by setting the root user to use password instead of `unix_socket`: https://stackoverflow.com/questions/69776205/disable-root-login-without-password-with-mariadb-10-5